tests/unit/s2n_send_multirecord_test.c

```diff
@@ -164,6 +164,32 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_stuffer_skip_write(&conn->out, 1));
             EXPECT_TRUE(s2n_should_flush(conn, buffer_size));
         }
+
+        /* Flush if buffer can't hold alert record */
```

Comment:
I understand this test based on the description from the PR. But it confused me at first. My first instinct was that we should bump it up to the size of a KeyUpdate message perhaps we should add a comment to clarify?

Reply:
Thanks for this comment :) While writing up the comment, I realized we don't actually need to account for alert sizes here. I added a unit test to prove it / enforce it.